node/network/protocol/src/lib.rs

```diff
@@ -438,6 +438,62 @@ pub mod v1 {
 	impl_try_from!(CollationProtocol, CollatorProtocol, CollatorProtocolMessage);
 }
 
+/// Unified annoyance cost and good behavior benefits.
+#[derive(Debug,Clone,Copy)]
+#[allow(missing_docs)]
+pub enum UnifiedReputationChange {
+	MajorAnnoy(&'static str),
+	MinorAnnoy(&'static str),
+	MajorAnnoyRepeated(&'static str),
+	MinorAnnoyRepeated(&'static str),
+	Malicious(&'static str),
+	BenefitMinorFirst(&'static str),
+	BenefitMinor(&'static str),
+	BenefitMajorFirst(&'static str),
+	BenefitMajor(&'static str),
+}
+
+impl UnifiedReputationChange {
+	/// Obtain the cost or benefit associated with
+	/// the enum variant.
+	const fn cost_or_benefit(&self) -> i32 {
+		match self {
+			Self::MinorAnnoy(_) => -100,
+			Self::MajorAnnoy(_) => -300,
+			Self::MinorAnnoyRepeated(_) => -200,
+			Self::MajorAnnoyRepeated(_) => -600,
+			Self::Malicious(_) => -1_000,
+			Self::BenefitMajorFirst(_) => 150,
+			Self::BenefitMajor(_) => 100,
+			Self::BenefitMinorFirst(_) => 15,
+			Self::BenefitMinor(_) => 10,
```

Comment:
I'd double check with the networking team and researches these are good numbers.

Reply:
These are really just the initial best guess ones based on the existing static values per subsystem, so they might very well be wrong @tomaka

Reply:
Currently the full range of an `i32` is used, and @tomaka recommended to increase those values significantly (order of magnitue `1e6` ) https://github.com/paritytech/substrate/blob/743accbe3256de2fc615adcaa3ab03ebdbbb4dbd/client/peerset/src/lib.rs#L48